tests/src/tests/tuple.spec.ts

```diff
@@ -160,7 +160,7 @@ export = () => {
 
 	it("should allow LuaTuples to have Array<> inside", () => {
 		function foo(): LuaTuple<[number, number, ...Array<string>] | []> {
-			return ([1, 2, "3"] as unknown) as LuaTuple<[number, number, ...Array<string>] | []>;
+			return [1, 2, "3"] as unknown as LuaTuple<[number, number, ...Array<string>] | []>;
```

Comment:
I believe these were intentionally parenthesised. You've probably run `prettier` which has inadvertently removed them?